src/bosh-director-core/lib/bosh/director/core/templates/job_template_renderer.rb

```diff
@@ -23,6 +24,13 @@ def render(spec)
         spec = remove_unused_properties(spec)
       end
 
+      if @release
```

Comment:
Can this conditional be removed? As far as I can think, templates should always be affiliated with a particular release.